packages/ckeditor5-image/src/imageinsert/imageinsertviaurlui.ts

```diff
@@ -40,6 +40,15 @@ export default class ImageInsertViaUrlUI extends Plugin {
 		return [ ImageInsertUI ] as const;
 	}
 
+	public init(): void {
+		this.editor.ui.componentFactory.add( 'menuBar:uploadUrl', locale => {
```

Comment:
I see you added this as a component but AFAICS, it is not used anywhere. So I think it should be used by `menuBarButtonViewCreator` right?

Additionally, there was a request from the team working on the builder to extract the toolbar button as a component as well, so it can be added directly to the toolbar. All image insert methods should have all corresponding components / integrations:

* toolbar button
* menu bar item
* integration with toolbar image insert dropdown
* integration with menu bar image insert menu